src/achievements.config.ts

```diff
@@ -21,6 +21,22 @@ export default AchievementSpec({
       { count: 30, title: 'PR Perfectionist', details: 'Reviewed 30 PRs' },
     ],
   },
+  'i18n-reviews': {
+    getCount: ({ reviews_by_category }) => {
+      if (!reviews_by_category) return 0;
+      let sum = 0;
+      for (const repo of Object.keys(reviews_by_category)) {
+        if (reviews_by_category[repo]['i18n']) 
+          sum += reviews_by_category[repo]['i18n'];
+      }
+      return sum;
+    },
+    achievements: [
+      { count: 1, title: 'Proofreader', details: 'Reviewed a i18n PR' },
+      { count: 10, title: 'Polyglot', details: 'Reviewed 10 i18n PRs' },
+      { count: 30, title: 'Rosetta Stone', details: 'Reviewed 30 i18n PRs' }
```

Comment:
Bumping up the thresholds here (and fixing `a` => `an` in the first badge description):
```suggestion
      { count: 1, title: 'Proofreader', details: 'Reviewed an i18n PR' },
      { count: 15, title: 'Polyglot', details: 'Reviewed 15 i18n PRs' },
      { count: 40, title: 'Rosetta Stone', details: 'Reviewed 40 i18n PRs' }
```